src/main/drivers/pwm_output.h

```diff
@@ -29,6 +29,26 @@
 #endif
 
 typedef enum {
+    DSHOT_CMD_MOTOR_STOP = 0,
+    DSHOT_CMD_BEEP1,
+    DSHOT_CMD_BEEP2,
+    DSHOT_CMD_BEEP3,
+    DSHOT_CMD_BEEP4,
+    DSHOT_CMD_BEEP5,
+    DSHOT_CMD_ESC_INFO,
+    DSHOT_CMD_SPIN_ONE_WAY, 
+    DSHOT_CMD_SPIN_OTHER_WAY, 
+    DSHOT_CMD_3D_MODE_OFF,
+    DSHOT_CMD_3D_MODE_ON, 
+    DSHOT_CMD_SETTINGS_REQUEST,
+    DSHOT_CMD_SAVE_SETTINGS, 
+    DSHOT_CMD_ROTATE_NORMAL = 20, //Blheli_S only command
+    DSHOT_CMD_ROTATE_REVERSE = 21,  //Blheli_S only command
```

Comment:
// 0-11 = SN, 12 = version, 13 = subversion (letter), 14 = esc type, 15=ESC rotationdirection, 16=esc 3dmode 
ESCInfos[PullESC][12] = ESCtempBuffer[13]; // 12 = 255 to tell that we use the new protocol
ESCInfos[PullESC][13] = ESCtempBuffer[14]; // additional letter
ESCInfos[PullESC][14] = ESCtempBuffer[15]; // ESC type
ESCInfos[PullESC][15] = ESCtempBuffer[16]; // rotation direction
ESCInfos[PullESC][16] = ESCtempBuffer[17]; // 3dmode
ESCInfos[PullESC][17] = ESCtempBuffer[18]; // unused for now
ESCInfos[PullESC][18] = ESCtempBuffer[19]; // unused for now